storage/innobase/srv/srv0start.cc

```diff
@@ -1264,13 +1264,13 @@ dberr_t srv_start(bool create_new_db)
 
 	if (!srv_read_only_mode) {
 		if (srv_innodb_status) {
-
+			ulong file_name_len = strlen(fil_path_to_mysql_datadir)
```

Comment:
the correct datatype returned by `strlen` is `size_t`, not `ulong`.  Using longs is discourages by the coding standard, and also this line made pull request fail on Windows builders.

Reply:
Thanks, I made these type adjustments.